README.md

```diff
@@ -1 +1,78 @@
 # testcafe-angular-selectors
+
+This plugin provides selector extensions that make it easier to test Angular applications with [TestCafe](https://github.com/DevExpress/testcafe/).
+
+## Install
+
+```
+npm install testcafe-angular-selectors
+```
+
+## Angular1 Selector extensions
+
+### byBinding
+Find elements by text binding. Does a partial match, so any elements can be returned.
```

Comment:
It is shorten phrase from https://github.com/angular/protractor/blob/master/lib/locators.ts#L102